_events/2022-07-05-Community-Call.md

```diff
@@ -1,13 +1,13 @@
 ---
 site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
 tags: training
-title: "Galaxy Community Call: Workflows, Planemo, BioBlend and tags to automate SARS-CoV-2 genome surveillance"
-starts: 2022-06-09
-ends: 2022-06-09
```

Comment:
This change does not belong into that PR I think.

Reply:
no I'm not sure how that change got there. can you skip it or should I modify it on my side?